library/ziti.c

```diff
@@ -571,17 +571,25 @@ static void session_cb(ziti_session *session, ziti_error *err, void *ctx) {
         }
 
         if (ztx->opts->refresh_interval > 0 && !uv_is_active((const uv_handle_t *) &ztx->refresh_timer)) {
+            ZITI_LOG(INFO, "refresh_interval set to [%d] ms", ztx->opts->refresh_interval * 1000);
```

Comment:
any reason log in ms instead of seconds?

Reply:
hunh. nope not really sure what happened there. i think i might have misinterpreted the variable at one point - i'll conver that over to seconds

Reply:
oh it's probably because of the next line: ```
            uv_timer_start(&ztx->refresh_timer, services_refresh, 0, ztx->opts->refresh_interval * 1000);```

Reply:
i changed it to seconds